src/test/java/net/sf/jabref/logic/citationstyle/CitationStyleGeneratorTest.java

```diff
@@ -43,4 +44,76 @@ public void testMissingCitationStyle() {
         assertEquals(expected, citation);
     }
 
+    @Test
+    public void testAsciiDocFormat() {
+        String expectedCitation = "[1] B. Smith, B. Jones, and J. Williams, ``Title of the test entry,'' __BibTeX Journal__, vol. 34, no. 3, pp. 45–67, Jul. 2016.\n";
+        BibEntry entry = TestEntry.getTestEntry();
+        String style = CitationStyle.getDefault().getSource();
+        CitationStyleOutputFormat format = CitationStyleOutputFormat.ASCII_DOC;
+
+        String actualCitation = CitationStyleGenerator.generateCitation(entry, style, format);
```

Comment:
I overlooked `CitationStyleToClipboardWorkerTest`. There, the multi-entry-tests are done. Everything allright 👍